### ISSUE TITLE
Not working : (

### ISSUE THREAD

**Issue by cnotte** [keeg/sketch-text-uppercase#1]
I installed this plug-in through Sketch Toolbox, followed the directions but can't get it to work.  Any suggestions?

**Comment by zeke** [comment]
Restarting Sketch.app did the trick for me. :)

**Comment by christiancummins** [comment]
Yup. It's not working for me either. even after restarting. I'm on yosemite. I'll try restarting the mac and see if it works.

**Comment by zeke** [comment]
Shortly after installing this plugin, I realized that the latest version of Sketch.app has this feature built in:

<img width="1280" alt="screen shot 2015-09-19 at 8 32 23 pm" src="https://cloud.githubusercontent.com/assets/2289/9978778/96964828-5f0d-11e5-8cd1-58aa4de65356.png">
